src/app/helptext/system/cloudcredentials.ts

```diff
@@ -15,6 +15,16 @@ export const helptext_system_cloudcredentials = {
     validation: [Validators.required]
   },
 
+  client_id: {
+    placeholder: T('OAuth Client ID'),
+    tooltip: T(''),
```

Comment:
@william-gr we need some starting point on the tooltips for these two fields.

Reply:
https://developers.google.com/identity/protocols/OAuth2